content/docs/introduction/roadmap.md

```diff
@@ -72,6 +72,15 @@ other long-term storage backends.
 
 GitHub issue: [#10](https://github.com/prometheus/prometheus/issues/10)
 
+**Improved staleness handling**
+
+Currently Prometheus considers a time series stale when it has no data points
+within the past 5 minutes. This prevents things like propogating timestamps
```

Comment:
For the issue of staleness in between queries, the whole issue of interpolation also comes up. If a time series has a gap of an hour, we probably want to drop results for it in the middle, but the behavior of still interpolating 5 minutes worth of stale data into this interval from both sides is probably also not ideal. Not exactly sure how it should behave in the ideal case.